client/src/+assets/landing.tsx

```diff
@@ -72,12 +71,11 @@ export const Landing: NextPage = () => {
             </UiText>
             <UiDivider type="s" />
             <UiText type="h4" variants={['secondary']}>
-              Shop
+              Subscribe
             </UiText>
             <UiDivider type="s" />
             <UiText type="p" variants={['detail']}>
-              Connect your wallet (Polygon or MetaMask) before adding the selected data sets to your
-              basket.
+            Connect your wallet (Polygon or Metamask) and purchase the Subscription you are interested in
```

Comment:
I think we should change this, polygon is not a wallet. Just something like connect your wallet (metamask) ?

Reply:
makes sense to me